src/Components/AppShell/AppShell.js

```diff
@@ -21,40 +21,14 @@ ElectronCookies.enable({
 })
 
 class AppShell extends Component {
-  constructor () {
-    super()
+  constructor (props) {
+    super(props)
```

Comment:
in the future, `super(...arguments)` is more future-proof

Reply:
Duly noted; however, the implemented way is consistent with the rest of the repo.